news/2025/2025-01-18-new-featured-artist-ntyn.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "New Featured Artist: ntyn"
+date: 2025-01-18 20:00:00 +0000
+---
+
+The time has finally come: **ntyn** is part of our Featured Artist catalogue!
+
+![](https://assets.ppy.sh/artists/456/header.jpg)
+
+Taiwanese producer **ntyn** isn't someone who sticks to a single style. He's an artist who specializes in a bunch of genres, switching between calm orchestral pieces and exciting pop tracks effortlessly.
+
+**6** songs (available as pre-timed `.osz` files) are officially part of [**ntyn**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/456). Pick out a song and start mapping already.
+
+Check everything out:
+
+### [ntyn - We Love Idol, and Master! (Chiptune)](https://assets.ppy.sh/artists/456/Songs/ntyn%20-%20We%20Love%20Idol%2C%20and%20Master!%20(Chiptune).osz)
+
+Here on osu!, his most recognizable tune is a chiptune rendition of [We Love Idol, and Master!](https://osu.ppy.sh/beatmapsets/2258940) hosted by [Daycore](https://osu.ppy.sh/users/5596337), showcasing just one side of **ntyn**'s multi-genre capabilities.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/456/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
```

Comment:
the video is usually before the subheadings

Reply:
i want to try this format when there's only 1 song in the video